db/sg_replicate_cfg.go

```diff
@@ -352,11 +352,23 @@ func (m *sgReplicateManager) StartReplication(config *ReplicationCfg) (replicato
 		return nil, fmt.Errorf("Unknown replication filter: %v", config.Filter)
 	}
 
-	rc.Direction = ActiveReplicatorDirection(config.Direction)
+	rc.Direction = config.Direction
 	if !rc.Direction.IsValid() {
 		return nil, fmt.Errorf("Unknown replication direction: %v", config.Direction)
 	}
 
+	// Set conflict resolver for pull replications
+	if rc.Direction == ActiveReplicatorTypePull {
```

Comment:
I think this needs to be set for a PushAndPull replication too.

This isn't being caught by TestActiveReplicatorPushAndPullConflict because that test is one level down from cbgt distributed SG-replications (it's creating the ActiveReplicator inside the test, rather than having SG create it based on config).

I think it raises a good point, now the config is wired up, and replication distribution working, we should see what tests we can move to be RT+ServerConfig based for the increased coverage.

```suggestion
	if rc.Direction == ActiveReplicatorTypePull || rc.Direction == ActiveReplicatorTypePushAndPull {
```